autoload/coiledsnake.vim

```diff
@@ -659,7 +667,11 @@ function! s:BufferWidth() abort "{{{1
         let signwidth = 0
     endif
 
+    if g:coiled_snake_explicit_sign_width != 0
+        let signwidth = g:coiled_snake_explicit_sign_width * 2
+    endif
+
```

Comment:
Can you merge this into the preceding if/elseif/else block that deals with the accounting for the sign column?

Reply:
![image](https://user-images.githubusercontent.com/56714626/89249923-1a3a6e80-d631-11ea-932e-e6eefc557f1b.png)
Like so?
But then it won't work for `neovim`, as there is an option like `set scl = auto:9` which means that the signcolumn shall expand till it can accomodate `9`  signs in a single line.

Actually, I meant that variable as an override over the value that `vim-coiled-snake` calculates.

Reply:
Sorry, this is what I meant:

```suggestion
    if g:coiled_snake_explicit_sign_width != 0
￼        let signwidth = g:coiled_snake_explicit_sign_width * 2
    elseif &signcolumn == 'yes'
        let signwidth = 2
    elseif &signcolumn == 'auto'
        " The `:sign place` output contains two header lines.
        " The sign column is fixed at two columns, if present.
        let signlist = execute(printf('sign place buffer=%d', bufnr('')))
        let signlist = split(signlist, "\n")
        let signwidth = len(signlist) > 2 ? 2 : 0
    else
￼        let signwidth = 0
￼    endif
```
I didn't know that neovim supports the colon syntax you used in the previous PR.  I wonder if that just a neovim-specific feature, or if vim supports it do but I missed it in the documentation.

Reply:
It is a neovim-specific feature. vim does not support it.